KarhooUISDK/Screens/CheckoutScreen/Components/CheckoutViews/KarhooExpandViewButton.swift

```diff
@@ -56,7 +47,7 @@ final class KarhooLearnMoreButton: UIButton {
         button.accessibilityIdentifier = KHBookingButtonViewID.button
         button.translatesAutoresizingMaskIntoConstraints = false
         button.addTarget(self, action: #selector(learnMorePressed), for: .touchUpInside)
-        button.accessibilityLabel = currentMode.title
+        button.accessibilityLabel = title
```

Comment:
Since `self` is an accessibility element anyway, this object (button) won't be triggered by voice-over. You can remove it, or leave it as it is.